manifests/create_bolt_workshop_targets.pp

```diff
@@ -37,40 +35,48 @@
     ],
   }
 
-  #Create the Linux target for teacher
```

Comment:
You are removing the deployment of the Bolt targets for the trainer, that is not acceptable

Reply:
@kreeuwijk hmm, strange, not sure what happened there. Resurrecting...